code/addons/links/tsconfig.json

```diff
@@ -1,8 +1,11 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "strict": true,
-    "skipLibCheck": true
+    "baseUrl": ".",
+    "paths": {
+      "storybook/internal/*": ["../../lib/cli/core/*"],
+      "react": ["../../node_modules/react"]
```

Comment:
Why is this necessary?

Reply:
Yeah.. please help me out removing it somehow...

TypeScript got things wrong, badly. and though this import referred to the `@storybook/addon-links/react` export.
Which is obviously very wrong, and TypeScript complained that it doesn't export any of the types needed.

I made TypeScript realize the correct path like this, but I'd love to remove this as well.